src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
+  }
```

Comment:
What is the magic numbers 20, 40, 50?
Create separate const for them
![2023-04-11_13h30_49](https://user-images.githubusercontent.com/61904995/231134230-0bf75652-b8f3-44f0-b7e6-936ed8fbd32c.png)

Do not use else statement
Return value if condition is true